### ISSUE TITLE
Missing instance for askLiterate

### ISSUE THREAD

**Issue by Shimuuar** [theam/haskell-do#95]
Not all common data types have `AskInliterate` instances. Non-exhaustive list

 - `()`
 - 'Integer`
 - `Maybe a`
 - `[a]`

This makes it impossible to run cells returning these types without wrapping them into `show`

**Comment by NickSeagull** [comment]
Yes, this is a problem of the `inliterate` library that we are using 😄

**Comment by ocramz** [comment]
Paging @glutamate ^^
